test/e2e/app-dir/app/next.config.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
     parallelServerCompiles: true,
     parallelServerBuildTraces: true,
     webpackBuildWorker: true,
+    appNavFailHandling: true,
```

Comment:
we should probably move this into a separate test so it doesn't ever accidentally skew existing `app-dir` tests